package.json

```diff
@@ -15,7 +15,7 @@
   ],
   "license": "Apache-2.0",
   "devDependencies": {
-    "@sugarcrm/eslint-config-sugarcrm": "git+https://github.com/sugarcrm/javascript",
+    "@sugarcrm/eslint-config-sugarcrm": "~1.0.0",
```

Comment:
Note: I used the tilde instead of the carat deliberately because I'm not sure what versioning policy we'll want to follow for the JS repo.